src/main/java/teammates/logic/core/CommentsLogic.java

```diff
@@ -49,15 +49,17 @@
     private static final FeedbackResponseCommentsLogic frcLogic = FeedbackResponseCommentsLogic.inst();
 
     public static CommentsLogic inst() {
-        if (instance == null)
+        if (instance == null) {
             instance = new CommentsLogic();
+        }
         return instance;
     }
     
     /************ CRUD ************/
 
-    public CommentAttributes createComment(CommentAttributes comment)
-            throws InvalidParametersException, EntityAlreadyExistsException, EntityDoesNotExistException {
+    public CommentAttributes createComment(CommentAttributes comment) throws InvalidParametersException,
+                                                                             EntityAlreadyExistsException,
+                                                                             EntityDoesNotExistException {
```

Comment:
The original is better. No benefit by stacking exceptions like this at the expense of extra lines.